app/models/dns-record.server.ts

```diff
@@ -126,3 +126,13 @@ export function getExpiredDnsRecords() {
     },
   });
 }
+
+/**
+ * This Fn gets the base data that is needed for reconciliation
+ * but does that for all records in the db. Should be no more than 10k
+ */
+export function getReconciliationData() {
+  return prisma.dnsRecord.findMany({
+    select: { username: true, subdomain: true, type: true, value: true },
```

Comment:
Any value in pre-sorting this at the db level?

Reply:
I don't think so, it should be rare that we handle multiple values for a single domain-type pair, and I'd not 100% trust that the sort result is the exact same between js sort() and mysql order by.

Reply:
in other cases sorting is not meaningful, we'll generate our internal structure